Bundler/Viewer/Unity/Assets/Scripts/Network/Scripts/MyNetworkManager.cs

```diff
@@ -241,7 +249,9 @@ private IEnumerator CheckForMigration()
                 }
 
             }
-            else if (migrationManager.disconnectedFromHost && migrationManager.oldServerConnectionId != -1)
+            else if (migrationManager != null && 
```

Comment:
check migrationManager != null once and add the other checks inside that condition?

Reply:
Fixed